codecov.yml

```diff
@@ -10,9 +10,13 @@ comment:
 coverage:
   precision: 1
   range: 97...98 # red -> yellow (the inside range) -> green
+  status:
+    patch:
+      default:
+        target: auto
+        threshold: 80%
```

Comment:
the PATCH check is too aggressive, lowering to 80. The important threshold is not PATCH it is PROJECT. Previously if we added 50 new lines of code and one of was not unit tested, the whole PR check would fail, that is too aggressive IMO. With this change "only" 4/5 lines need tested for the PATCH to pass the PR chech.